linux/network_device_ops/network_device_test.yml

```diff
@@ -3,8 +3,10 @@
 ---
 - block:
     - include_tasks: ../setup/test_setup.yml
-      vars:
-        deploy_router_vm: True
+
+    # Prepare router VM, vSwitch and portgroup
+    - include_tasks: ../../common/network_testbed_setup.yml
+      when: router_vm_deployed is undefined or not router_vm_deployed | bool
```

Comment:
Same as above, `network_testbed_setup.yml` can be executed as late as it can, here can be executed before `network_status_validate.yml`.

Reply:
Move this before `- include_tasks: network_status_validate.yml`.